CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+- **Breaking**: converted sources to TypeScript. Exports are now ES6 modules with the `default` property where a single object was exported before.
```

Comment:
My intent was to use native TS as much as possible but I can easily drop it here and go back to `export =` exports